jest.config.js

```diff
@@ -27,6 +27,7 @@ module.exports = {
   collectCoverage: true,
   collectCoverageFrom: [
     '**/packages/**/*.{js,jsx}',
+    '!**/packages/test/**',
     '!**/packages/**/story.jsx',
```

Comment:
Since we are changing this file I'm wondering if we can add this as well 😅 
```suggestion
    '!**/packages/**/story.jsx',
    '!**/packages/**/story_test.jsx',
```